saleor/graphql/product/mutations/products.py

```diff
@@ -196,7 +196,7 @@ class AttributeValueInput(InputObjectType):
         required=True, description='Value of an attribute.')
 
 
-class ProductInput(graphene.InputObjectType):
+class ProductPropertiesInput(graphene.InputObjectType):
```

Comment:
@oldPadavan I would do a similar change as in your other PR - let's change it here to `ProductInput` and have dedicated `ProductCreateInput` only for product creation.